CHANGELOG.md

```diff
@@ -15,7 +15,9 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
 
 ### Removed Features
 
-### Deprecated Fatures
+### Deprecated Features
+
+- ROX-24169: API token authentication has been deprecated by Red Hat OpenShift Cluster Manager. The corresponding cloud source integration now uses service accounts for authentication.
```

Comment:
Can we link to their announcement?

Reply:
Sadly their is no public announcement as per the OCM team.